avocado/utils/network/interfaces.py

```diff
@@ -313,6 +318,36 @@ def save(self, ipaddr, netmask):
                       'IPV6_DEFROUTE': 'yes'}
         if current_distro.name == 'SuSE':
             ifcfg_dict.pop('BOOTPROTO')
+
+        if self.if_type == 'Bond':
+            # Update master and slave cfg for bond
+            ifcfg_dict.update({'BONDING_MASTER': 'yes'})
```

Comment:
`update` is usually a good idea if you have more than one variable to set.  In this case, I find the following preferrable:

```python
ifcfg_dict['BONDING_MASTER'] = 'yes'
```